BLSQ/iaso
# IA-632 leaflet

Upgrade leaflet (1.9.3) and react-leaflet(3.2.5).

Related JIRA tickets : IA-632

## Self proofreading checklist


## Changes

- Remove map reducer 
- upgrade every maps to support new versions of packages
- rewrite of custom zoom bar
- provide some docs for frontend and especially for maps
- use only one tile layer component

## How to test

Run `npm ci` first.

Go trough every maps in Iaso and check that everything is ok:
- GeoJsonMapComponent => visible in org unit history for an org unit with a shape
- MarkerMapComponent => visible in instance details with a geo location
- AssignmentsMap => planning
- ListMap => entities /beneficiaries
- InstancesMap => instances list
- OrgUnitsMap => org unit list
- OrgUnitMap => org unit detail
- OrgUnitChildrenMap => registry
- CalendarMap => Polio/calendar

## Notes

Bluesquare component as been upgraded [here](https://github.com/BLSQ/bluesquare-components/pull/121)
